src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={`${
+              tab.id === selectedTabId
+              || (!tabs.find(tabWithTheID => tabWithTheID.id === selectedTabId)
+              && tab.id === tabs[0].id)
+                ? 'is-active'
+                : ''
+            }`}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => {
+                if (tab.id !== selectedTabId) {
+                  onTabSelected(tab);
+                }
+              }}
+            >
+              {tab.title}
+            </a>
+          </li>
+        ))}
+      </ul>
+    </div>
+
+    <div className="block" data-cy="TabContent">
+      {tabs.find(tab => tab.id === selectedTabId)
```

Comment:
you are repeating logic from a function `getCurrentTab that could be reused for this case
like this:
```
const selectedTab = getCurrentTab(selectedTabId);
```
```
{selectedTab ? (
  <div className="block" data-cy="TabContent">{selectedTab.content}</div>
) : (
 <p>Tab is not selected</p>
)}
```